src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -351,88 +351,128 @@ public struct S
         }
 
         [Fact]
-        public void ReadOnlyMethod_OverrideBaseMethod()
```

Comment:
>ReadOnlyMethod_OverrideBaseMethod [](start = 20, length = 33)

was this test removed?

Reply:
It was migrated to CodeGenReadOnlyStructsTests because I added IL verification